modular_skyrat/modules/goofsec/code/department_guards.dm

```diff
@@ -196,6 +196,8 @@
 	shoes = /obj/item/clothing/shoes/jackboots
 	head =  /obj/item/clothing/head/helmet/blueshirt/skyrat
 	suit = /obj/item/clothing/suit/armor/vest/blueshirt/skyrat
+	suit_store = /obj/item/melee/baton/security/loaded/departmental/science
```

Comment:
put these in the locker, suit storage items will bug out if you're wearing a loadout suit.